src/main/java/by/antonsh/lesson10/Ex29.java

```diff
@@ -0,0 +1,21 @@
+package by.antonsh.lesson10;
+
+import java.util.ArrayList;
+import java.util.ListIterator;
+import java.util.Random;
+import java.util.stream.Collectors;
+
+/*
+Создать список оценок учеников с помощью ArrayList,
+заполнить рандом. Удалить неудовлетворительные оценки
+ */
+public class Ex29 {
+    public static void main(String[] args) {
+        Random random=new Random();
+        ArrayList<Integer> grade=new ArrayList<Integer>(
+                random.ints(15,3,10).boxed().collect(Collectors.toList()));
+        System.out.println(grade);
+        grade= (ArrayList<Integer>) grade.stream().filter(a->a>=7).collect(Collectors.toList());
+        System.out.println(grade);
```

Comment:
Please, sometimes format code to right tabulation. Ctrl+Alt+L. Also in new ArrayList<Integer> write <Integer> not necessary, only new ArrayList<>. Diamond syntax

Reply:
Ok,thks